### PR TITLE
Restore old transcription workflow

### PR DESCRIPTION
**Pull request by Qubad786**
[EDUCATOR-2408](https://openedx.atlassian.net/browse/EDUCATOR-2408)

This restores old transcription flow when credentials were stored with VEDA course.

### CODE REVIEW

**Comment by Qubad786** [comment]
@muhammad-ammar @mushtaqak @irfanuddinahmad may you guys give it a look?

File: control/veda_deliver.py [hunk 1]
```diff
@@ -629,6 +635,72 @@ def start_3play_transcription_process(self, encoded_file):
                 self.video_query.studio_id,
             )
 
+    def _CIELO24_UPLOAD(self):
+        """
+        Note: This is part of the old flow which was deprecated when transcript phase 1 was reased.
+        """
+        # TODO: this must be removed once existing 3rd party credentials are migrated according
```
Comment (mushtaqak) [inline control/veda_deliver.py]: @Qubad786 Would there be any issue now we are reverting this change?

File: control/veda_deliver.py [hunk 2]
```diff
@@ -128,6 +129,11 @@ def run(self):
         u1.encode_size = self.video_proto.filesize
         u1.save()
 
+        # TODO: Warning! this shall be removed once 3rd party credentials
```
Comment (mushtaqak) [inline control/veda_deliver.py]: I think, we should create a follow-up ticket that tracks this and when 3rd party Automated Video Transcription phases are rolled out, we would need to clean this again.

@irfanuddinahmad FYI

File: control/veda_deliver.py [hunk 3]
```diff
@@ -5,10 +5,10 @@
 
 """
 import datetime
+import ftplib
 import logging
 import shutil
 from os.path import expanduser
-import sys
```
Comment (mushtaqak) [inline control/veda_deliver.py]: We are using `sys` in the code below.

File: control/old_veda_deliver_cielo.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        print str(r4.status_code) + ' : Cielo24 Status Code'
+        return ast.literal_eval(r4.text)['TaskId']
+
+
+def main():
+    pass
+
+
+if __name__ == "__main__":
+    sys.exit(main())
```
Comment (mushtaqak) [inline control/old_veda_deliver_cielo.py]: `sys` is not imported

File: control/veda_deliver.py (hunk 2 above)
Comment (Qubad786) [reply control/veda_deliver.py]: we have it: https://openedx.atlassian.net/browse/EDUCATOR-1321

File: control/old_veda_deliver_cielo.py (hunk 4 above)
Comment (Qubad786) [reply control/old_veda_deliver_cielo.py]: its coming from `from control_env import *`

File: control/veda_deliver.py (hunk 3 above)
Comment (Qubad786) [reply control/veda_deliver.py]: its coming from `from control_env import *`

File: control/veda_deliver.py (hunk 1 above)
Comment (Qubad786) [reply control/veda_deliver.py]: nope, but we have to make the credentials are migrated.

**Pull request merged**